src/pages/PlaceList/PlaceList.jsx

```diff
@@ -0,0 +1,303 @@
+    if (type === undefined) {
+      const result = [];
+      Object.entries(data).forEach((filterData) => filterData[1] && result.push(filterData.join('=')));
+      return result && result.join('&');
+    } else {
+      const result = [];
+      data.forEach((filterData) => result.push(`${type}=${filterData}`));
+      return result && result.join('&');
+    }
+  };
+
+  return (
+    <PlaceListComponent>
+      <ListContainer>
+        <h4>300개 이상의 숙소</h4>
+        <h1>지도에서 선택한 지역의 숙소</h1>
+        <PlaceFilterPlaceList
+          houseTypte={houseTypte}
+          setHouseType={setHouseType}
+          modalState={modalState}
+          setModalState={setModalState}
+          facility={facility}
+          setfacility={setfacility}
+        />
+        <span className='notice'>여행 날짜와 게스트 인원수를 입력하면 1박당 총 요금을 확인할 수 있습니다.</span>
+        <div className='layoutLine'></div>
+        {placeList.length ? (
+          placeList.map((el, index) => {
+            return (
+              <PlaceItem key={index}>
+                <img className='placeImg' src={el.propertyImages} />
+
+                <div className='placeContent'>
+                  <div className='contentTitle'>
+                    <div>
+                      <div className='subTitle'>아직 모르는 데이터</div>
+                      <div className='title'>{el.propertyName}</div>
+                    </div>
+                    <div className='placeDibs'>{true ? <AiOutlineHeart /> : <AiFillHeart />}</div>
+                  </div>
+                  <div className='layoutLine'></div>
+                  <div className='option'>
+                    <span>최대인원 6명 ' 침실 2개 ' 침실 2개 ' 욕실 1개</span>
+                    <span>주방 ' 무료 주차 공간 ' 무선 인터넷 ' 난방</span>
+                  </div>
+                  <div className='placeValue'>
+                    <div className='placeGrade'>
+                      <AiFillStar className='plceGradeIcon' />
+                      <span className='plceGradeNumber'>4.64</span>
+                    </div>
+                    <div className='placePrice'>
+                      <span className='price'>{parseInt(el.price).toLocaleString()}</span>
+                      <span className='perMonth'>/월</span>
+                    </div>
+                  </div>
+                </div>
+              </PlaceItem>
+            );
+          })
+        ) : (
+          <div>데이터가 없습니다</div>
+        )}
+        <div className='pageNate'>
+          <ReactPaginate
+            pageCount={Math.ceil(50 / 10)}
+            pageRangeDisplayed={10}
+            marginPagesDisplayed={0}
+            breakLabel={'다다음'}
+            previousLabel={'이전'}
+            nextLabel={'다음'}
+            onPageChange={(cnt) => {
+              setCurrentPage(cnt.selected * 10);
+            }}
+            containerClassName={'pagination-ul'}
+            activeClassName={'currentPage'}
+            previousClassName={'pageLabel-btn'}
+            nextClassName={'pageLabel-btn'}
```

Comment:
중괄호는 필요한 경우에만 사용해주세요
```suggestion
            containerClassName='pagination-ul'
            activeClassName='currentPage'
            previousClassName='pageLabel-btn'
            nextClassName='pageLabel-btn'
```

Reply:
아하 그렇군요 알겠습니다